app/src/main/java/com/example/pocketcloset/RandomizedView.java

```diff
@@ -26,6 +27,15 @@
 
 public class RandomizedView extends AppCompatActivity {
     protected List<Clothing> allClothing;
+    List<Clothing> randomTops;
```

Comment:
Could we abstract this duplication in some way? For example:

```
class ClothingSelection {
    List<Clothing> items;
    ClothingAdapter adapter;
    RecyclerView view;
    @Nullable SnapHelper snapHelper;
}
```

And then create an array and generalize the code that populates the data structures? The above pseudo code may be insufficient because it doesn't include the clothing type. Not immediately sure off hand if that should be a property, or if it would make sense to use as a key in a map.